src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -36,7 +37,7 @@
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-final class IriConverter implements IriConverterInterface
+final class IriConverter implements IriToItemConverterInterface, ItemToIriConverterInterface
```

Comment:
I think you need to check in the whole code base where IriConverterInterface is used.

Reply:
Pretty sure that removing an interfave from a class is a BC break following SF rules...